frontend/src/components/dashboard/data/Demo.vue

```diff
@@ -196,7 +196,9 @@ const changeCompare = (val: string) => {
           },
           color: {
```

Comment:
wondering if we should replace this block, conditional on whether the plot has any defined vars in a variable in scope to this component.

Reply:
not a blocker-
I recommend using a map to replace the entire `color` section/block if we have to update more than one variable to get custom behavior; this way we put the architecture for changing behavior in place even if we're not going to use the logic immediately. I would prefer doing this over leaving documentation around which vars to change if someone wants to make a customization.